go.mod

```diff
@@ -11,28 +12,36 @@ replace (
 require (
 	github.com/agiledragon/gomonkey v2.0.2+incompatible
 	github.com/agiledragon/gomonkey/v2 v2.4.0
```

Comment:
This file includes lots of function to operate(create/update) resources like service, and in nsx-operator test, will be replaced by apply yaml.
Those operation function may be deleted.

Reply:
which file ?

Reply:
the file with prefix k8s,  it seems you have deleted it